src/lib/jitter/jitter.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+* Jitter utility functions
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <span>
+
+#include "jitter.h"
+
+#ifdef BOTAN_HAS_JITTER
```

Comment:
This guard is not necessary - this file is not compiled unless the module is included